app/api/v2/models/users.py

```diff
@@ -0,0 +1,49 @@
+from app.api import Tools 
+import uuid
+from app.api.v2.models import conn, cur, cur2
+
+tools = Tools()
+# all_users = tools.all_users()
+
+
+
+class UserDetails():
+
+    @staticmethod
+    def register(username, email, phone_number, password, confirm_password):
+
+        user = tools.user_exists(email, phone_number, username)
+        if user == False:
+            hash_pass = tools.generate_hash(password)
+            if hash_pass:
+                #If The user does not exist validate the information provided and store the user in the database
+                validate = tools.validate_user_info(username, email, phone_number, password, confirm_password)
+
+                if validate == True:
```

Comment:
You can also use `if validate` which turns to `True` when the conditions are met